pyproject.toml

```diff
@@ -28,6 +28,7 @@ dependencies = [
     "pandas==2.2.3",
     "numpy==2.1.2",
     "mmh3==5.0.1",
+    "pip-licenses",
```

Comment:
I don't think we shoud add it here as it is not a direct dependency on the project. I would add it in the `RUN` command of the dockerfile instead